src/content_cao.cpp

```diff
@@ -1572,209 +1572,216 @@ void GenericCAO::processMessage(const std::string &data)
-		m_tx_size.Y = 1.0 / m_prop.spritediv.Y;
-
-		if(!m_initial_tx_basepos_set){
-			m_initial_tx_basepos_set = true;
-			m_tx_basepos = m_prop.initial_sprite_basepos;
-		}
-
-		if ((m_is_player && !m_is_local_player) && m_prop.nametag == "")
-			m_prop.nametag = m_name;
-
-		expireVisuals();
-	}
-	else if(cmd == GENERIC_CMD_UPDATE_POSITION)
-	{
-		// Not sent by the server if this object is an attachment.
-		// We might however get here if the server notices the object being detached before the client.
-		m_position = readV3F1000(is);
-		m_velocity = readV3F1000(is);
-		m_acceleration = readV3F1000(is);
-		if(fabs(m_prop.automatic_rotate) < 0.001)
-			m_yaw = readF1000(is);
-		else
-			readF1000(is);
-		bool do_interpolate = readU8(is);
-		bool is_end_position = readU8(is);
-		float update_interval = readF1000(is);
-
-		// Place us a bit higher if we're physical, to not sink into
-		// the ground due to sucky collision detection...
-		if(m_prop.physical)
-			m_position += v3f(0,0.002,0);
-
-		if(getParent() != NULL) // Just in case
-			return;
-
-		if(do_interpolate)
-		{
-			if(!m_prop.physical)
-				pos_translator.update(m_position, is_end_position, update_interval);
-		} else {
-			pos_translator.init(m_position);
-		}
-		updateNodePos();
-	}
-	else if(cmd == GENERIC_CMD_SET_TEXTURE_MOD) {
-		std::string mod = deSerializeString(is);
-		updateTextures(mod);
-	}
-	else if(cmd == GENERIC_CMD_SET_SPRITE) {
-		v2s16 p = readV2S16(is);
-		int num_frames = readU16(is);
-		float framelength = readF1000(is);
-		bool select_horiz_by_yawpitch = readU8(is);
-
-		m_tx_basepos = p;
-		m_anim_num_frames = num_frames;
-		m_anim_framelength = framelength;
-		m_tx_select_horiz_by_yawpitch = select_horiz_by_yawpitch;
-
-		updateTexturePos();
-	}
-	else if(cmd == GENERIC_CMD_SET_PHYSICS_OVERRIDE) {
-		float override_speed = readF1000(is);
-		float override_jump = readF1000(is);
-		float override_gravity = readF1000(is);
-		// these are sent inverted so we get true when the server sends nothing
-		bool sneak = !readU8(is);
-		bool sneak_glitch = !readU8(is);
-
-
-		if(m_is_local_player)
-		{
-			LocalPlayer *player = m_env->getLocalPlayer();
-			player->physics_override_speed = override_speed;
-			player->physics_override_jump = override_jump;
-			player->physics_override_gravity = override_gravity;
-			player->physics_override_sneak = sneak;
-			player->physics_override_sneak_glitch = sneak_glitch;
-		}
-	}
-	else if(cmd == GENERIC_CMD_SET_ANIMATION) {
-		// TODO: change frames send as v2s32 value
-		v2f range = readV2F1000(is);
-		if (!m_is_local_player) {
-			m_animation_range = v2s32((s32)range.X, (s32)range.Y);
-			m_animation_speed = readF1000(is);
-			m_animation_blend = readF1000(is);
-			// these are sent inverted so we get true when the server sends nothing
-			m_animation_loop = !readU8(is);
-			updateAnimation();
-		} else {
-			LocalPlayer *player = m_env->getLocalPlayer();
-			if(player->last_animation == NO_ANIM)
+	switch (cmd) {
```

Comment:
Notice: It is allowed to remove a whole level of indentation after `switch (cmd) {`
https://www.kernel.org/doc/Documentation/CodingStyle (Chapter 1: Indentation)